packages/tools/pluggable-widgets-tools/bin/mx-scripts.js

```diff
@@ -77,6 +77,10 @@ function getRealCommand(cmd, toolsRoot) {
         case "test:e2e:ts":
         case "test:e2e:js":
             return `wdio ${join(toolsRoot, "test-config/wdio.conf.js")}`;
+        case "test:e2e:native":
+            return `node ${join(toolsRoot, "scripts/e2e.js")} --native`;
+        case "test:e2e:native:dev":
+            return `detox test --configuration ios.simulator`;
```

Comment:
we probably need better separation ios/android, because i can imagine that locally you want to run both